doc/user_guide.md

```diff
@@ -33,6 +33,16 @@ Through the use of [coroutines](http://en.wikipedia.org/wiki/Coroutine) this MQT
 * The library can send and receive simultaneously on a single socket.
 * Communication requirements of your application will not interfere with the usability of your device.
 
+### Secure File Transfer (SFT)
+
+The Xively C Client comes pre-built with functionality to communicate with the [Xively SFT Service](https://developer.xively.com/v1.0/docs/how-to-securely-transfer-files-to-my-devices).  
+
+* Devices can be easily updated in the field with new firmware and support files.
+* Data is downloaded in the background, no need to create a second connection to a new service.
+* Download and update status is reported by the Xively C Client to the SFT service automatically, helping you to monitor the progress of a software deployment across your fleet of devices.
```

Comment:
well phrased as usual 👍

Reply:
Thanks, took me a while to carve this out actually.